src/SFA.DAS.Commitments.Infrastructure/Services/ApprenticeshipEvents.cs

```diff
@@ -108,7 +107,9 @@ private static Events.Api.Types.ApprenticeshipEvent CreateEvent(Commitment commi
                 TransferSenderId = commitment.TransferSenderId,
                 TransferSenderName = commitment.TransferSenderName,
                 TransferApprovalStatus = (Events.Api.Types.TransferApprovalStatus?) commitment.TransferApprovalStatus,
-                TransferApprovalActionedOn = commitment.TransferApprovalActionedOn
+                TransferApprovalActionedOn = commitment.TransferApprovalActionedOn,
+                StoppedOnDate = paymentStatus == PaymentStatus.Withdrawn ? effectiveFrom : apprenticeship.StopDate,
```

Comment:
When pausing or stopping an apprentice (as indicated by paymentStatus), effectiveFrom contains the pause/stop date.

Reply:
Is this true? Isn't the stop date added by the employer explicitly?

Reply:
Yes it's true :)
I'm not sure what you mean by your other questions here. The stop date *is* set explicitly by the employer in the UI, my comment relates to what happens in the code.